modules/internal/ChapelIteratorSupport.chpl

```diff
@@ -473,55 +475,136 @@ module ChapelIteratorSupport {
     return _toStandalone(x.these(), (...args));
   }
 
+  // arrays: can lead fast followers, can produce fast followers
+  // domains: can lead fast followers, doesn't produce fast followers
+  // other iterators: cannot lead fast followers, doesn't produce fast followers
```

Comment:
Should we note that "other iterators" won't prevent fast followers from triggering if all other iterands are fast?

Reply:
Nevermind, I see that is covered below